js/app/pages/Form.html

```diff
@@ -258,15 +254,15 @@
     }
     app.modal.handleStudentNote(student);
   },
-  handleItemNoteInput: function(event) { // keydown.key == 'Enter' or blur
+  /* handleItemNoteInput: function(event) { // keydown.key == 'Enter' or blur
```

Comment:
if this function is getting killed, let's just delete it

Reply:
removed on latest commit